platform/mbed_wait_api.h

```diff
@@ -46,9 +46,9 @@ extern "C" {
  * int main() {
  *     while (1) {
  *         heartbeat = 1;
- *         wait(0.5);
+ *         ThisThread::sleep_for(500);
```

Comment:
```suggestion
 *         wait_us(500);
```

Reply:
Why wait_us?

Reply:
From my comment above - this is supposed to be an example of use of the wait APIs, so changing it to use the sleep API doesn't make sense.